packages/ui/ViewManager/TransitionGroup.js

```diff
@@ -183,6 +183,9 @@ class TransitionGroup extends Component {
 		this.keysToLeave = [];
 		this.keysToStay = [];
 		this.groupRefs = {};
+		this.nodeRef = createRef();
+		this.refNodeId = '#transition#group#';
+		this.refNode = (<div data-transitiongroup-target={this.refNodeId} ref={this.nodeRef} style={{display: 'none'}} />);
```

Comment:
I prefer to have this in render function.

Reply:
okay~